spec/Vehicle/Vehicle.vspec

```diff
@@ -136,6 +141,11 @@ VehicleIdentification.VehicleSpecialUsage:
   type: attribute
   description: Indicates whether the vehicle has been used for special purposes, like commercial rental, driving school.
 
+VehicleIdentification.VehicleExteriorColor:
```

Comment:
Do we need to more explicit on what we expect this signal to contain, for example stating that it shall be the OEM name of the color, rather than color code, and that the names used are not standardized. For example, a possible name could be  "Carnelian red metallic".

Reply:
I think it makes sense

Reply:
I'll provide a better description of the property